hypertrace-core-graphql-log-event-schema/src/main/java/org/hypertrace/core/graphql/log/event/schema/LogEventSchema.java

```diff
@@ -0,0 +1,25 @@
+package org.hypertrace.core.graphql.log.event.schema;
+
+import graphql.annotations.annotationTypes.GraphQLDataFetcher;
+import graphql.annotations.annotationTypes.GraphQLField;
+import graphql.annotations.annotationTypes.GraphQLName;
+import graphql.annotations.annotationTypes.GraphQLNonNull;
+import java.util.List;
+import org.hypertrace.core.graphql.common.schema.arguments.TimeRangeArgument;
+import org.hypertrace.core.graphql.common.schema.results.arguments.filter.FilterArgument;
+import org.hypertrace.core.graphql.common.schema.results.arguments.order.OrderArgument;
+import org.hypertrace.core.graphql.common.schema.results.arguments.page.LimitArgument;
+import org.hypertrace.core.graphql.common.schema.results.arguments.page.OffsetArgument;
+import org.hypertrace.core.graphql.common.schema.results.arguments.space.SpaceArgument;
+import org.hypertrace.core.graphql.log.event.fetcher.LogEventFetcher;
+
+public interface LogEventSchema {
+  String LOG_EVENTS_QUERY_NAME = "";
+
+  @GraphQLField
+  @GraphQLNonNull
+  @GraphQLName(LOG_EVENTS_QUERY_NAME)
+  @GraphQLDataFetcher(LogEventFetcher.class)
+  LogEventResultSet logEvents(
+      @GraphQLName(FilterArgument.ARGUMENT_NAME) List<FilterArgument> filterBy);
```

Comment:
limit, time range, offset etc. - all the same params as span

Reply:
We will be querying logs based on spanId/TraceID, do we need to support these params?

Reply:
Limit, offset and sort yes (because we'll be paging and sorting on log tables). Time range, I could be convinced we can defer on - we'll need it once we support general log querying, but it does make querying more of a pain when querying by ID. We usually have it anyway for perf reasons so all pinot segments don't need to be searched, but if there's no perf concern, I'm ok deferring.

Reply:
Yes, without any additional index, it will need a scan to all segments. And, by default, the segments are created using timeSpecs. So, I would also suggest that we can add a `between` clause for filtering. So, this is quite the same as our `spans` request.

Reply:
Makes sense, will support between, orderBy, offset, limit.